src/core/Akka/Event/LoggingBus.cs

```diff
@@ -188,12 +188,14 @@ private void AddLogger(ActorSystemImpl system, Type loggerType, LogLevel logLeve
             object response = null;
             try
             {
-                response = askTask.Result;
```

Comment:
Calling `.Result` throws `AggregateException` instead of the expected `AskTimeoutException` exception.